Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -426,19 +436,21 @@ - (BOOL)startGraph:(NSError **)error
     return YES;
 }
 
-- (BOOL)updateOutputSampleRate:(OCTToxAVSampleRate)rate error:(NSError **)error
+- (BOOL)updateOutputSampleRate:(OCTToxAVSampleRate)rate channels:(OCTToxAVChannels)channels error:(NSError **)error
 {
+    DDLogVerbose(@"%@ updateOutputSampleRate:%ul channels:%ul", self, rate, channels);
+
     UInt32 bytesPerSample = sizeof(SInt16);
 
     AudioStreamBasicDescription asbd = {0};
     asbd.mSampleRate = rate;
     asbd.mFormatID = kAudioFormatLinearPCM;
     asbd.mFormatFlags = kLinearPCMFormatFlagIsSignedInteger;
-    asbd.mChannelsPerFrame = kNumberOfChannels;
-    asbd.mBytesPerFrame = bytesPerSample * kNumberOfChannels;
+    asbd.mChannelsPerFrame = channels;
+    asbd.mBytesPerFrame = bytesPerSample * channels;
     asbd.mBitsPerChannel = kBitsPerByte * bytesPerSample;
     asbd.mFramesPerPacket = kFramesPerPacket;
-    asbd.mBytesPerPacket = bytesPerSample * kNumberOfChannels;
+    asbd.mBytesPerPacket = bytesPerSample * channels;
```

Comment:
`kNumberOfChannels` constant is now used only for input channels. Can you please rename it to `kNumberOfInputChannels`?